### ISSUE TITLE
collapse fails silently if fastq is missing or not an absolute path

### ISSUE THREAD

**Issue by diekhans** [BrooksLabUCSC/flair#170]
Run FLAIR with a missing FASTQ or not specifying relative path to the fastq, such as:

```
 python3 ../flair/flair.py collapse -g ../../GRCh38_full_analysis_set_plus_decoy_hla.fa -r NA12878-cDNA_All_Guppy_4.2.2.fastq -q NA12878-cDNA_All_Guppy_4.2.2.hg38.bed12 -t 128
```
results in no error message and a silent failure, with an empty output BED.

If one keeps intermediate files, one sees in tmp*.firstpass.mm2_stderr that there is a minimap error:

```

  [M::mm_idx_stat::149.190*3.41] distinct minimizers: 6418271 (50.26% are singletons); average occurrences: 23.494;
  ERROR: failed to open file '../NA12878-cDNA_All_Guppy_4.2.2.fastq'
  [M::main] Version: 2.17-r941
  [M::main] CMD: minimap2 -a -t 128 -N 4 flair.collapse.firstpass.fa ../NA12878-cDNA_All_Guppy_4.2.2.fastq
  [M::main] Real time: 149.883 sec; CPU: 509.480 sec; Peak RSS: 23.563 GB

```

**Comment by Jeltje** [comment]
Thanks!
Fixed in commit 58eb5d7e61596fa351b5178ab3b4f7066446ed77
Will be part of all releases >1.6.1

**Comment by diekhans** [comment]
❤️
